### PR TITLE
change more GL to GLES2 includes when USE_GLES=1 is used

### PR DESCRIPTION
**Pull request by krnlyng**
does this break some functionality? if so how can i fix it?

### CODE REVIEW

**Comment by littleguy77** [comment]
I have not tried building GlideHQ for GLES yet.  We simply omit it from mupen64plus-ae.
@paulscode @xperia64 do you have any time to give this a shot?

File: src/GlideHQ/tc-1.1+/s2tc/s2tc_libtxc_dxtn.cpp [hunk 1]
```diff
@@ -257,6 +259,7 @@ void tx_compress_dxtn(GLint srccomps, GLint width, GLint height,
 				blkaddr += dstRowDiff;
 			}
 			break;
+#ifndef USE_GLES
 		case GL_COMPRESSED_RGBA_S3TC_DXT3_EXT:
 			dstRowDiff = dstRowStride >= (width * 4) ? dstRowStride - (((width + 3) & ~3) * 4) : 0;
 			/*      fprintf(stderr, "dxt3 tex width %d tex height %d dstRowStride %d\n",
```
Comment (Narann) [inline src/GlideHQ/tc-1.1+/s2tc/s2tc_libtxc_dxtn.cpp]: Commented code here.

**Comment by Narann** [comment]
I would like to have feedbacks from glide64mk2 OpenGL ES 2 users before merge this.

**Comment by littleguy77** [comment]
I can confirm, these are the changes needed for the GLES codepath to build.  I had to cherry-pick this commit when I was working on the boost dependency removal.

**Pull request merged**